core/README.md

```diff
@@ -54,8 +42,25 @@ lerna bootstrap
 The `lerna run` command runs the given script in every package referenced in the `lerna.json` file.
 ```bash
 lerna run bundle
+// or non-minified and watcher:
+lerna run bundle-develop
 ```
 
+**Link manually and build**
+
+If you do not want to use lerna.
```

Comment:
If you do not want to use Lerna, create manual links and build.  Follow these steps to manually link local modules:

Reply:
Removed this block, as we decided yesterday just require to use lerna instead of confusion by to many options.